modules/assim.sequential/R/Analysis_sda.R

```diff
@@ -277,20 +273,27 @@ GEF<-function(setting,Forecast,Observed, H, extraArg, nitr=50000, nburnin=10000,
   #### from the interval matrix
   y.ind <- as.numeric(Y > interval[,1])
   y.censored <- as.numeric(ifelse(Y > interval[,1], Y, 0))
-  
+  browser()
```

Comment:
This seems to be a debugging leftover. If it's meant to be there, it seems like it would be good to put in within an optional flag, e.g.
```
if(debug){
    browser()
}
```

Reply:
I still use arguments to pass the data to the sda function.

Reply:
I don't understand -- your response about passing data to the function as an argument doesn't seem to have anything to do with the use of `browser()`